core/src/main/java/nu/yona/server/subscriptions/entities/UserPrivate.java

```diff
@@ -31,6 +32,9 @@
 
 	private static final String DECRYPTION_CHECK_STRING = "Decrypted properly#";
 
+	@Column(nullable = true)
```

Comment:
When I left that out I got a NOT NULL constraint violation from HSQLDB when trying to insert it. I tried to initialize the field with `private int touchVersion = 1;` but that didn't work. If you have a different suggestion that works please let me know.